src/main/java/io/antmedia/AntMediaApplicationAdapter.java

```diff
@@ -211,6 +212,8 @@ public class AntMediaApplicationAdapter  extends MultiThreadedApplicationAdapter
 
 	private Random random = new Random();
 
+	private JSONParser jsonParser = new JSONParser();
```

Comment:
AKAIK, JsonParser is not thread-safe. We've encountered weird errors about it. For now, create local variable for that.

Reply:
done